site.yml

```diff
@@ -48,11 +50,18 @@ asciidoc:
     idprefix: ''
     idseparator: '-'
     experimental: ''
+    # to be deleted after the 10.8 branch has been deployed
     latest-version: 10.7
     latest-download-version: 10.7.0
     previous-version: 10.6
     current-version: 10.7
-    page-version: 10.7
+    # to be deleted
```

Comment:
Lots of references have been adjusted to use `latest-server-version` etc.
So why does this comment say "to be deleted"?

Reply:
Atm, in site.yml we have variables for the server release but the name itself does not reflect it. When looking to desktop which is built atm via this repo, we do now also have not only master but branch releases. To have the right branch used during the build for the desktop part, we need new variables. I took the chance to add new once for server which reflect the name so we know what it is good for. Imagine the future where the build base and server are in seperate repos. When 10.8 is created which is a clone of master, we have 2 identical branches  and 10.7 which is slightly different, 10.6 is archived and out. At that moment we can savely delete the „old“ variables as they are no longer be used in any branch, not in the definition (site.yml) nor in any documentation. 10.6 has too many dependencies on the old variables and is in addition too different for an easy backport. Therefore I have decided not to go that path. When this PR is merged, the proper BP to 10.7 is made and the new 10.8 branch is created, 10.6 is archived, we can safely remove the old varaibles from the 3 branches which I tried to mark 😎

Reply:
Now I think I understand - this comment is the end of the "to be deleted" section.
I thought it indicated that the next line is "to be deleted"